lib/rest-adapter.js

```diff
@@ -586,7 +609,7 @@ function RestMethod(restClass, sharedMethod) {
   this.notes = sharedMethod.notes;
   this.documented = sharedMethod.documented;
 
-  var methodRoutes = getRoutes(sharedMethod);
+  var methodRoutes = getRoutes(sharedMethod, options);
```

Comment:
Usually, when a constructor accepts an `options` argument, then we store it in the class instances, so that `options` can be accessed from any member method. See [RemoteObjects constructor](https://github.com/strongloop/strong-remoting/blob/5b89fb2ab79a77c5b4d25b4009e8033951eb4852/lib/remote-objects.js#L63). OTOH, apparently SharedClass and SharedMethod don't store options directly, so we are not consistent in this.

> I don't see the `options` being stored on the `RestClass` instance.

`options` is a new argument added by your pull request. Since you are adding it, you can also store it in the instance.

This is not a big deal though. If you prefer to keep passing options via function arguments instead of `this.options`, then I can live with that too.